README.md

```diff
@@ -205,7 +205,6 @@ There is a little difference between __arguments__ and __parameters__:
 ## Higher-Order Functions (HOF)
 
 A function that takes a function as an argument and/or returns a function, basically we can treat functions as a value.
-In Python every function/method is a Higher-Order Function.
```

Comment:
This is not the way, you don't need to remove the line!!

Beyond functional jargon, it's functional jargon __using__ Python. So, this information is important!
Just reformulating the phrase should be enough.

Reply:
Well, it's your text, I am not sure what you were trying to say, but the current wording is false.
Maybe "Python supports higher-order functions."?

Reply:
Sorry for the long delay!!
What about, "In Python every function/method can be a Higher-Order Function"??

Reply:
👍